.swiftlint.yml

```diff
@@ -1,14 +1,13 @@
 # Rules: https://realm.github.io/SwiftLint/rule-directory.html
+# Config file learn more: https://github.com/realm/SwiftLint#configuration=
+
+excluded: # paths or files to ignore during linting. Takes precedence over `included`.  
+  - .build
+  - Package.swift
```

Comment:
Instead of `include`, use `exclude` so that there is a smaller chance of lint not running against code in our project that we add over time.